.github/workflows/build.yaml

```diff
@@ -155,3 +155,36 @@ jobs:
 
       - name: Run instrumented tests
         run: scripts/android/run_instrumented_tests.sh
+
+  release:
+    needs: [desktop, android-linux, android-osx]
+    if: startsWith(github.ref, 'refs/tags/v')
+
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - java: 8
+            api: 29
+            ndk: 21.1.6352462
+
+    name: android/linux-java${{ matrix.java }}-api${{ matrix.api }}-ndk${{ matrix.ndk }}
```

Comment:
I think we should give it some name that differs from "regular" android-linux jobs; maybe just "release" or "aar" or "publish" or something like this? IIUC, we will have only one job that builds and publishes AAR, right?

Reply:
yeah, only one job.
I've changed name and removed matrix